modules/services/safeeyes.nix

```
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.safeeyes;
+
+in {
+  meta.maintainers = [ maintainers.rosuavio ];
+
+  options = {
+    services.safeeyes = {
+      enable = mkEnableOption "the safeeyes OSGI service";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.safeeyes;
+      };
```

Can do something like

```suggestion
      package = mkPackageOption pkgs "safeeyes" { };
```